docs/Android/allowed-apps.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: page
+title: Intelligence SDK Allowed Apps
+hide:
+  #- navigation
+  - toc
+---
+
+## Intelligence SDK Allowed Apps
+
+The WS1 SDK Custom Settings KVP `IntelSDKAllowedApps` (Allowed Apps lists) is a JSON Array which can be used to control which apps can report DEX Data.
```

Comment:
Can we add a line here which says SDK control config is nothing but UEM custom SDK settings and where it could be fetched from?